src/Keycloak.tsx

```diff
@@ -0,0 +1,108 @@
+import Keycloak from 'keycloak-js';
+import React from 'react';
+
+import { Auth, AuthContext } from '@rhoas/app-services-ui-shared';
+
+export let keycloak: Keycloak.KeycloakInstance | undefined;
+
+/**
+ * Get keycloak instance
+ *
+ * @return an initiated keycloak instance or `undefined`
+ * if keycloak isn't configured
+ *
+ */
+export const setKeycloakInstance = async () => {
+  if (!keycloak) await init();
+};
+
+/**
+ * Initiate keycloak instance.
+ *
+ * Set keycloak to undefined if
+ * keycloak isn't configured
+ *
+ */
+export const init = async () => {
+  try {
+    /* Keycloak configuration using OB authentication.
+     * Thiese params will change after sso is implemented.
```

Comment:
Typo `these`

Reply:
Whoops. Thanks for spotting it! Fixed now.